packages/flutter_tools/lib/src/devfs.dart

```diff
@@ -566,228 +494,27 @@ class DevFS {
           ? fs.path.relative(pathToReload, from: projectRootPath)
           : pathToReload,
         );
-        if (!dirtyEntries.containsKey(entryUri)) {
```

Comment:
Why was this `if` removed, I'm curious?

Reply:
I'm not sure how this could ever be false - this entryUri is for the dill file and at this point we have only added assets to the dirtyEntries.